### DISCUSSION TITLE
Clarification on what to VCS-ignore and what to commit?

### DISCUSSION THREAD

**Discussion by paul-griffith** [shannah/jdeploy#41] [Q&A] [answered]
Setting up jdeploy on a small project for which it seems absolutely perfect (https://github.com/paul-griffith/kindling)
I've been trying to work through the overall excellent documentation, and one sticking point for me is what to do with auto-generated files. I'm not sure which files are actually necessary, or exactly why they're created. For instance, from the repository root, running `jdeploy publish` generates two distinct folder trees, which seem to have some duplicate items:
![image](https://user-images.githubusercontent.com/39345262/156078515-c57d5851-eeeb-4cf1-91b6-7a2e1571e13f.png)

Do I ignore both subfolders and all their contents, and only commit the root `package.json` and any icons I have? Is it more efficient to commit any of these files?

**Comment by shannah** [comment, marked as answer]
Both the jdeploy-bundle and jdeploy directories can be kept out of version control.

The jdeploy directory will contain build outputs from creating bundles. (if you have the _bundles_ or _installers_ properties defined) in the jdeploy object of the package.json

The jdeploy-bundle directory includes all of the files that are published to npm when you run "publish".  Both of these are transient build artifacts that will be regenerated each time you build - so no need to have them in version control.
